scripts/script.js

```diff
@@ -0,0 +1,127 @@
+    currentInfo: document.querySelector('.weather-current-info'),
+    direction: document.querySelector('.icon-direction-pointer'),
+    wind: document.querySelector('.weather-wind'),
+    pressure: document.querySelector('.weather-pressure'),
+    humidity: document.querySelector('.weather-humidity'),
+    dew: document.querySelector('.weather-dew'),
+    visibility: document.querySelector('.weather-visibility'),
+    loading: document.querySelector('.is-loading')
+};
+Object.freeze(elements)
+
+function userGeolocation() {
+    navigator.geolocation.getCurrentPosition(
+        (data) => {
+            const lat = data.coords.latitude;
+            const lon = data.coords.longitude;
+            findCityByGeolocation(lat, lon)
+        },
+        (error) => {
+            gettingError(error.message)
+            if (!window.localStorage.getItem("userCity")) {
+                findCityByName(cityList[0])
+            } else {
+                findCityByName(window.localStorage.getItem('userCity'))
+            }
+        }
+    )
+}
+
+function createCityList(list) {
+    for (let i = 0; i < list.length; i++) {
+        const { cityList } = elements
+        const city = createDOMElement('li', 'weather-city-city')
+        city.innerHTML = list[i]
+        city.setAttribute("data-for", list[i])
+        cityList.append(city)
+    }
+}
+
+function toggleList() {
+    const { cityList } = elements
+    cityList.classList.toggle('hidden')
+}
+
+function animation() {
+    const { loading } = elements
+    loading.classList.remove('is-loading')
+}
+
+function selectedCityUpdate() {
+    const { citySelected } = elements
+    citySelected.innerHTML = window.localStorage.getItem('userCity')
+}
+
+async function findCityByName(cityName) {
+    const response = await fetch(`https://api.openweathermap.org/data/2.5/weather?q=${cityName}&appid=${apiKey}&units=metric`)
+    const data = await response.json()
+    if (!response.ok) {
+        responseErrorChecker(data)
+    }
+    updater(data)
+}
+
+async function findCityByGeolocation(lat, lon) {
+    const response = await fetch (`https://api.openweathermap.org/data/2.5/weather?lat=${lat}&lon=${lon}&appid=${apiKey}&units=metric`)
+    const data = await response.json()
+    if (!response.ok) {
+        responseErrorChecker(data)
+    }
+    updater(data)
+}
+
+function updater(data) {
+    const city = new CityInfo(data)
+
+    updateCity(city)
+    selectedCityUpdate()
+}
+
+function updateCity(city) {
+    elements.images.forEach( ele => {
+        const imageElement = document.createElement('img')
+        imageElement.setAttribute('src', `https://openweathermap.org/img/wn/${city.icon}@2x.png`)
+        ele.innerHTML = ""
+        ele.append(imageElement)
+    })
+    city.secondState = firstCharToUpperCase(city.secondState)
+    elements.direction.style.transform = `rotate(${city.windDeg}deg)`
+
+    elements.temperatures.forEach( ele => ele.innerHTML = `${city.temp}° C`)
```

Comment:
bad parameter naming. Make correct abbreviations of words
```suggestion
    elements.temperatures.forEach( elem => elem.innerHTML = `${city.temp}° C`)
```